templates/tutorialv2/view/content.html

```diff
@@ -131,7 +131,7 @@ <h2 class="subtitle">
                     {% for help in content_helps %}{% if not forloop.first %}{% if forloop.last %}{% trans ' et ' %}{% else %}{% trans ', ' %}{% endif %}{% endif %}un {{ help.title|lower }}{% if forloop.last %}{% trans '.' %}{% endif %}{% endfor %}
 
                     {% if not can_edit %}
-                        {% blocktrans with plural=content.authors.count|pluralize_fr %}
+                        {% blocktrans with plural=content.redacting_authors.count|pluralize_fr %}
```

Comment:
Étant donné que `redacting_authors` est une liste, je ne crois pas que `.count` fonctionne. Si ça ne fonctionne pas, alors ceci devrait fonctionner : 
```suggestion
                        {% blocktrans with plural=content.redacting_authors|length|pluralize_fr %}
```